internal/api/cluster_installation.go

```diff
@@ -297,10 +297,15 @@ func handleRunClusterInstallationExecCommand(c *Context, w http.ResponseWriter,
 	}
 
 	args := append([]string{fmt.Sprintf("./bin/%s", command)}, clusterInstallationExecSubcommand...)
-	output, err := c.Provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, args...)
+	output, execErr, err := c.Provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, args...)
 	if err != nil {
-		c.Logger.WithError(err).Error("failed to execute command")
+		c.Logger.WithError(err).Error("failed to prepare command execution")
 		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	if execErr != nil {
+		c.Logger.WithError(execErr).Error("failed to execute command")
+		w.WriteHeader(http.StatusConflict)
```

Comment:
I couldn't find a status code that really fit, but this one seemed pretty okay. Let me know what you think.

Reply:
Perhaps 502 would fit better? I guess it is hard to determine whether the error was client error or server error, but 502 would indicate not provisioner error but the on in downstream component. 1/5 on that, I am fine with 409 either if you prefer it.

Reply:
Yeah, 502 is a bit better, but avoiding anything 500 or above for now as that is how our opsgenie alerts determine when to fire or not.